frontend/plugins/content_ui/src/modules/cms/custom-fields/types/customFieldTypes.tsx

```
@@ -0,0 +1,99 @@
+  IconChevronDown,
+  IconCircleCheck,
+  IconFile,
+  IconLink,
+  IconListDetails,
+  IconNumbers,
+  IconPhoto,
+  IconSquareCheck,
+  IconTable,
+  IconTextScan2,
+  IconTextSize,
+} from '@tabler/icons-react';
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)